policyengine_us/parameters/gov/states/oh/tax/income/credits/lump_sum_distribution/base_amount.yaml

```diff
@@ -0,0 +1,10 @@
+description: Ohio multiplies the filer's expected remaining life in years by the following amount when calculating the lump sum distribution credit.
+values:
+  2021-01-01: 50
+metadata:
+  unit: currency-USD
+  period: year
+  label: Ohio lump sum distribution credit base amount 
+  reference:
+    - title:  Ohio 2021 Instructions for Filing Original and Amended Individual Income Tax (IT 1040) and  School District Income Tax (SD 100)
```

Comment:
add the law as a reference here: 
https://law.justia.com/codes/ohio/2022/title-57/chapter-5747/section-5747-055/